### PR TITLE
Add more style customization options to RichTextEditor

### PR DESCRIPTION
**Pull request by viniciusaa**
### Implementation Screenshot or GIF
Screenshot from 2022-01-05 14-29-58

### Notes:
N/A

### CODE REVIEW

File: src/components/RichTextEditor/RichTextEditor.css [hunk 1]
```diff
@@ -9,14 +9,25 @@
...
   color: var(--button-color-is-active);
 }
 
 .menu-bar-container {
+  align-items: var(--menu-bar-align-items, flex-start);
+  background-color: var(--menu-bar-background-color, --structure-color-white);
+  border-radius: var(--menu-bar-border-radius, 0);
   display: flex;
   flex-wrap: wrap;
+  height: var(--menu-bar-height, fit-content);
```
Comment (paulohfev) [inline src/components/RichTextEditor/RichTextEditor.css:28]: @viniciusaa let's avoid using `fit-content` on the height or width. It's not widely supported yet and is known for causing some issues in other browsers.

**Review by paulohfev — CHANGES_REQUESTED** [review]

File: src/components/RichTextEditor/RichTextEditor.css (hunk 1 above)
Comment (viniciusaa) [reply src/components/RichTextEditor/RichTextEditor.css:28]: Any suggestions on what I should replace it with?

Comment (paulohfev) [reply src/components/RichTextEditor/RichTextEditor.css:28]: @viniciusaa in this case, I would advise using the 100% value, if the intent is to make sure the component fits the entire width or height of whatever container it is nested under. Please make sure you double-check for side-effects though.

File: src/components/RichTextEditor/RichTextEditor.css [hunk 2]
```diff
@@ -9,14 +9,25 @@
...
+}
+
 .is-active {
   background: var(--button-background-color-is-active);
   color: var(--button-color-is-active);
 }
 
 .menu-bar-container {
+  align-items: var(--menu-bar-align-items, flex-start);
+  background-color: var(--menu-bar-background-color, --structure-color-white);
```
Comment (paulohfev) [inline src/components/RichTextEditor/RichTextEditor.css:24]: @viniciusaa can you please a correction in this line. The secondary line should have the `var` keyword. It should be written as: `var(--menu-bar-background-color, var(--structure-color-white));`

**Review by paulohfev — CHANGES_REQUESTED** [review]

**Review by paulohfev — APPROVED** [review]

**Pull request merged**